changelog

```diff
@@ -1,3 +1,240 @@
+      * Fixed an issue where scanning a ship's cargo or outfits could cause the scan dialog to appear twice. (@warp-core)
+      * Selecting a new system on the map now always clears the system orbit data, instead of only doing so when the newly selected system has been visited. (@TomGoodIdea)
+      * Switching between two engines that use the same engine flare sprite but at different scales will now properly update the engine sprite scale. (@TomGoodIdea)
+      * Overlays like the planet labels are no longer stepped while the player is landed. This was causing planet labels to appear in the wrong position when landed. (@tibetiroka)
+      * NPC actions now properly trigger if a ship receives two simultaneous events. (@warp-core)
+      * Disabling and enabling plugins is now properly handled with the new plugin metadata file. (@quyykk)
+      * Long plugin names are now truncated to the appropriate length in the plugins preferences panel. (@warp-core)
+      * Travel panels that become invalid due to any reason (such as by system changes caused by events) are now cleared. (@warp-core)
+      * A valid CargoHold is now always returned when getting planetary storage, preventing segfaults. (@quyykk)
+      * Fixed a race condition that could occur when drawing planet labels, causing them to flicker. (@quyykk)
+      * Corrected the use of SDL_GetPrefPath in Files.cpp. (@quyykk)
+  * Game content:
+    * New content:
+      * New mission where you answer a distress call in Korath space. (@UnorderedSigh, @williaji)
+      * More human spaceport news. (@dorbarker, @tibetiroka)
+      * Added new humanitarion aid jobs to human space. (@dorbarker)
+      * Added a warning for when you've upset Unfettered sympathizers. (@MasterOfGrey)
+      * New mission where you take war refugees to Humanika. (@dseomn)
+      * New pre-war missions that help lead peoples to the start of the Free Worlds campaign if they're traveling elsewhere in the galaxy. (@danaris)
+      * Added more high-end passenger mission to human space. (@Anarchist2)
+    * Mission changes:
+      * Mentioned in the conversation and description of "Deal Change 2" that you need to hail the destination planet in order to land. (@bene-dictator)
+      * The player can now ask Tomek for combat advice prior to the first major Free Worlds battle. (@warp-core)
+      * Revised the smuggler arc of the Hai Reveal storyline. (@UnorderedSigh)
+      * Bounty jobs now provide their payment the moment the bounty fleet is destroyed, instead of requiring that the player return to the offer location of the job to receive pay. (@Anarchist2)
+      * Improved the wording in the description of "FW Bounty 1". (@m0ctave, @EjoThims)
+      * Added the mission destination to the description of "Remnant: Cognizance 7". (@lumbar527)
+      * The Heliarch jump drive jobs now use the <payment> text replacement instead of spelling out the payment. (@Arachi-Lover)
```

Comment:
If this is referring to the recent PR
```suggestion
      * The Heliarch jump drive jobs now use the <payment> text replacement instead of spelling out the payment. (@Arachi-Lover, @tibetiroka )
```
as the changes were mine.